engine/class_modules/sc_shaman.cpp

```diff
@@ -5272,6 +5280,58 @@ struct thundercharge_t : public shaman_spell_t
   }
 };
 
+// ==========================================================================
+// Fae Transfusion - Night Fae Covenant
+// ==========================================================================
+
+struct fae_transfusion_damage_t : public shaman_spell_t
+{
+  fae_transfusion_damage_t( shaman_t* player, const std::string& options_str )
+    : shaman_spell_t( "fae_transfusion_", player, player->find_spell( 328928 ), options_str )
+  {
+    aoe        = 4;
+    ground_aoe = background = true;
+    dynamic_tick_action     = true;
+    base_tick_time          = 500_ms;
+    dot_duration            = 3_s;
+  }
+
+  void impact( action_state_t* state ) override
+  {
+    shaman_spell_t::impact( state );
+  }
+};
+
+struct fae_transfusion_t : public shaman_spell_t
+{
+  fae_transfusion_damage_t* transfusion;
+  event_t* area_of_effect;
+
+  fae_transfusion_t( shaman_t* player, const std::string& options_str )
+    : shaman_spell_t( "fae_transfusion", player, player->covenant.fae_transfusion, options_str ),
+      transfusion( new fae_transfusion_damage_t( player, options_str ) )
```

Comment:
The secondary damage spell should not be constructed with any options. Options are only ever relevant for spells that go into the APL.